src/menu.c

```diff
@@ -435,14 +443,16 @@ int gophermap(state *st, char *mapfile, int depth)
 		}
 	}
 
+CLOSE_FP:
```

Comment:
I think this either needs to be renamed to CLEANUP or the comment below needs to be changed to "Close file pointers", the comment seems a bit strange for the goto name.

Reply:
I removed the comment, because I think the code and the label name are already pretty clear.